src/main/java/it/gov/pagopa/payment/dto/mapper/TransactionCreationRequest2TransactionInProgressMapper.java

```diff
@@ -0,0 +1,32 @@
+package it.gov.pagopa.payment.dto.mapper;
+
+import it.gov.pagopa.payment.dto.qrcode.TransactionCreationRequest;
+import it.gov.pagopa.payment.enums.Status;
+import it.gov.pagopa.payment.model.TransactionInProgress;
+import java.util.function.Function;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TransactionCreationRequest2TransactionInProgressMapper
+    implements Function<TransactionCreationRequest, TransactionInProgress> {
+
+  @Override
+  public TransactionInProgress apply(TransactionCreationRequest transactionCreationRequest) {
+    return TransactionInProgress.builder()
+        .acquirerId(transactionCreationRequest.getAcquirerId())
+        .acquirerCode(transactionCreationRequest.getAcquirerCode())
+        .amount(transactionCreationRequest.getAmount())
+        .amountCurrency(transactionCreationRequest.getAmountCurrency())
+        .merchantFiscalCode(transactionCreationRequest.getMerchantFiscalCode())
+        .callbackUrl(transactionCreationRequest.getCallbackUrl())
+        .idTrxAcquirer(transactionCreationRequest.getIdTrxAcquirer())
+        .idTrxIssuer(transactionCreationRequest.getIdTrxIssuer())
+        .initiativeId(transactionCreationRequest.getInitiativeId())
+        .mcc(transactionCreationRequest.getMcc())
+        .senderCode(transactionCreationRequest.getSenderCode())
+        .vat(transactionCreationRequest.getVat())
+        .trxDate(transactionCreationRequest.getTrxDate())
+        .status(Status.CREATED)
+        .build();
```

Comment:
fill also operationType with the constant "00" (let's put this value in a Constant class, together with a constant for the refund's operationType "01")

fill also the operationTypeTranscoded with the enum value CHARGE (copy the enum from the reward-calculator repository)
(operationType string representation could be a field of this enum)